packages/template-retail-react-app/app/ssr.js

```diff
@@ -34,6 +37,21 @@ const options = {
 
 const runtime = getRuntime()
 
+// const clientId = process?.env?.SLAS_PRIVATE_CLIENT_ID
+const clientId = getConfig().app.commerceAPI.parameters.clientId
+const secret = process?.env?.SLAS_PRIVATE_CLIENT_SECRET
+const encodedSlasCredentials = Buffer.from(`${clientId}:${secret}`).toString(
+    'base64'
+)
+
+let slasTarget
+getConfig().ssrParameters.proxyConfigs.forEach((config) => {
+    if (config.path == 'api') {
+        slasTarget = config.host
+    }
+})
```

Comment:
I wonder if there was a cleaner way of doing this? Lets think about how we are going to describe doing this to customers switching to private from an existing private client.